src/main/java/edu/cornell/kfs/sys/service/impl/CountryServiceImpl.java

```diff
@@ -90,6 +99,21 @@ public String findCountryNameByCountryCode(String countryCode) {
         }
     }
     
+    @Override
+    public List<String> findCountryCodesByCountryName(String countryName) {
+        if (isBlank(countryName)) {
+            return List.of();
+        }
+        QueryByCriteria criteria = QueryByCriteria.Builder.fromPredicates(
+                PredicateFactory.equalIgnoreCase(CUKFSPropertyConstants.Country.NAME, countryName),
```

Comment:
Table SH_CNTRY_T contains active rows where country names are in mixed case as well as all uppercase.
Do we need to make this query case insensitive?
Should leading and trailing white space also be addressed?

Reply:
This `equalIgnoreCase` predicate already makes the search case-insensitive. That's why I had set up the name-based searches to use KualiCo's criteria API and the `CriteriaLookupService`.

As for the leading and trailing whitespace, the calling code already deals with that, but I'll go ahead and also add that here, in case we have future areas of the code that need to use this method.